### DISCUSSION TITLE
Feature Request: Wikilinks [[link]] Support

### DISCUSSION THREAD

**Discussion by braden-w** [withastro/roadmap#437] [Proposal]
I was wondering if Astro intends to support wikilink syntax [[]] (e.g. [[Wikipedia]]), as commonly used in wikis and other collaborative writing platforms to create links to other pages within the same site. I would greatly benefit from the ability to create internal links more quickly, and this would be especially helpful for users of Obsidian, Logseq, Roam Research, etc.

Currently, Astro does not support this feature, and I was wondering if this is planned for the future.

**Comment by jasikpark** [comment]
i bet there's a remark plugin for it :p

**Comment by stereobooster** [comment]
Related https://github.com/withastro/starlight/discussions/1113

**Reply by stereobooster** [reply]
**UPD** https://astro-digital-garden.stereobooster.com/recipes/wikilinks/

**Comment by manunamz** [comment]
Here's an example astro setup for anyone who's interested:

https://github.com/wikibonsai/astro-bloomz

You can also turn on `[[wikilink]]` support for your parser of choice:

https://github.com/wikibonsai/remark-wikirefs
https://github.com/wikibonsai/markdown-it-wikirefs
https://github.com/wikibonsai/marked-wikirefs
